contrib/mesos/pkg/executor/executor.go

```diff
@@ -415,9 +415,18 @@ func (k *Executor) handleChangedApiserverPod(pod *api.Pod) {
 	}
 
 	oldPod := k.pods[task.podName]
+	if oldPod == nil {
+		// TODO(jdef) should sent a TASK_FAILED or TASK_LOST here
```

Comment:
could probably push to a DelayQueue, keyed on podID (with object replacement) with a maxAge == launchTimeout and just retry periodically.